src/app/public/modules/action-hub/types/action-hub-data.ts

```diff
@@ -0,0 +1,9 @@
+import { SkyLink } from '../../link-list/types/link';
+import { SkyActionHubNeedsAttention } from '../../needs-attention/types/action-hub-needs-attention';
+
+export type SkyActionHubData = {
```

Comment:
We will need JSDocs here in order to fully populate the docs.

Example: https://github.com/blackbaud/skyux-lookup/blob/master/src/app/public/modules/country-field/types/country.ts